### PR TITLE
Added overview and other additional details for Bamba

### PR DESCRIPTION
**Pull request by divya-kumari32**
Added overview, update Model inference card and added config

### CODE REVIEW

File: docs/source/en/model_doc/bamba.md [hunk 1]
```diff
@@ -16,51 +16,44 @@ rendered properly in your Markdown viewer.
...
+model = AutoModelForCausalLM.from_pretrained("ibm-fms/Bamba-9B")
+tokenizer = AutoTokenizer.from_pretrained("ibm-fms/Bamba-9B")
+
+message = ["I am an LLM and my name is "]
+inputs = tokenizer(message, return_tensors='pt', return_token_type_ids=False)
+response = model.generate(**inputs, max_new_tokens=100, do_sample=True, top_k=50, top_p=0.95)
+print(tokenizer.batch_decode(response, skip_special_tokens=True)[0])
+```
+
+This model was contributed by [ani300]https://github.com/ani300) and [fabianlim]https://github.com/fabianlim) .
```
Comment (lchu6) [inline docs/source/en/model_doc/bamba.md]: missing `(` ?

**Review by lchu6 — COMMENTED** [review]
LG, left a minor comment.

File: docs/source/en/model_doc/bamba.md [hunk 2]
```diff
@@ -16,51 +16,44 @@ rendered properly in your Markdown viewer.
...
+model = AutoModelForCausalLM.from_pretrained("ibm-fms/Bamba-9B")
+tokenizer = AutoTokenizer.from_pretrained("ibm-fms/Bamba-9B")
+
+message = ["I am an LLM and my name is "]
+inputs = tokenizer(message, return_tensors='pt', return_token_type_ids=False)
+response = model.generate(**inputs, max_new_tokens=100, do_sample=True, top_k=50, top_p=0.95)
+print(tokenizer.batch_decode(response, skip_special_tokens=True)[0])
+```
+
+This model was contributed by [ani300](https://github.com/ani300) and [fabianlim](https://github.com/fabianlim).
```
Comment (lchu6) [inline docs/source/en/model_doc/bamba.md]: maybe say "This HF implementation is contributed by", rather than "This model" to avoid confusion.

**Review by lchu6 — COMMENTED** [review]
Left another minor.

**Review by lchu6 — APPROVED** [review]

File: docs/source/en/model_doc/bamba.md (hunk 1 above)
Comment (divya-kumari32) [reply docs/source/en/model_doc/bamba.md]: Resolved

File: docs/source/en/model_doc/bamba.md (hunk 2 above)
Comment (divya-kumari32) [reply docs/source/en/model_doc/bamba.md]: resolved

**Pull request merged**